.changeset/silent-experts-sip.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-astro': major
```

Comment:
Why is this a `major`? I think it should be a `patch` because **if fixes** a bug. Do you agree?